src/test/java/racingcar/service/RacingCarsServiceTest.java

```diff
@@ -0,0 +1,96 @@
+package racingcar.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import racingcar.controller.dto.NamesRequestDto;
+import racingcar.controller.dto.RacingCountRequestDto;
+import racingcar.exception.IllegalCarNameException;
+import racingcar.exception.RaceNotCountException;
+import racingcar.vo.RaceProcessResponse;
+import racingcar.vo.RaceWinnerResponse;
+import strategy.MoveStrategy;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.IntStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * @author heebg
+ * @version 1.0
+ * @date 2019-11-22
+ */
+class RacingCarsServiceTest {
+    private List<String> names;
+    private int raceCount = 5;
+    private RacingCarsService service;
+    private MoveStrategy moveStrategy;
+
+    @BeforeEach
+    void setUp() {
+        names = Arrays.asList("hee", "bong", "hi");
+        service = new RacingCarsService();
+        NamesRequestDto namesRequestDto = new NamesRequestDto(names);
+        moveStrategy = () -> true;
+        service.createCars(namesRequestDto, moveStrategy);
+
+        RacingCountRequestDto countRequestDto = new RacingCountRequestDto(raceCount);
+        service.createRacingCars(countRequestDto);
+    }
+
+    @Test
+    void createCars() {
+        NamesRequestDto requestDto = new NamesRequestDto(names);
+        service.createCars(requestDto, moveStrategy);
```

Comment:
정상 흐름임을 알려줄 수 있도록 assertDoesNotThrow 단언 메서드를 사용하는 것은 어떤가요?

```suggestion
        assertDoesNotThrow(() -> service.createRacingCars(requestDto));
```

Reply:
훌륭..! 반영했습니다!